### PR TITLE
Intelligence listing - new content

### PR DESCRIPTION
**Pull request by ka0ula**

### CODE REVIEW

**Review by CharlesLR-sekoia — APPROVED** [review]

**Review by Darkheir — CHANGES_REQUESTED** [review]
It seems that the data model is different between the TIP and the CTI:
* https://storage.gra.cloud.ovh.net/v1/AUTH_741841e1c7da4c50a76152cbc2609bd1/sekoiaio-documentation-previews/668/tip/features/data_model/
* https://storage.gra.cloud.ovh.net/v1/AUTH_741841e1c7da4c50a76152cbc2609bd1/sekoiaio-documentation-previews/668/cti/features/data_model/


The new data model page you created is indeed way better than the old one

**Comment by ka0ula** [comment]
I thought that the two pages were similar @Darkheir :/ @otetard can you help with this one?

**Comment by Darkheir** [comment]
> I thought that the two pages were similar @Darkheir :/ @otetard can you help with this one?

I can do it.

**Comment by ka0ula** [comment]
> > I thought that the two pages were similar @Darkheir :/ @otetard can you help with this one?
> 
> I can do it.

would be awesome! Thanks

**Review by Darkheir — APPROVED** [review]
Made the change if its OK for you it is for me :-)

File: _shared_content/intelligence_center/data_model.md [hunk 1]
```diff
@@ -1,15 +1,50 @@
 # Data Model
```
Comment (ka0ula) [inline _shared_content/intelligence_center/data_model.md:1]:
```suggestion
# Data Models
```

**Pull request merged**